pkgs/applications/office/paperless-ngx/default.nix

```diff
@@ -37,17 +39,21 @@ let
   # https://github.com/NixOS/nixpkgs/issues/298719
   # https://github.com/paperless-ngx/paperless-ngx/issues/5494
   python = python3.override {
-    packageOverrides = self: super: {
-      uvicorn = super.uvicorn.overridePythonAttrs (oldAttrs: {
-        version = "0.25.0";
-        src = fetchFromGitHub {
-          owner = "encode";
-          repo = "uvicorn";
-          rev = "0.25.0";
-          hash = "sha256-ng98DTw49zyFjrPnEwfnPfONyjKKZYuLl0qduxSppYk=";
-        };
-      });
-    };
+    packageOverrides = lib.composeManyExtensions [
+      packageOverrides
+
+      (final: prev: {
+        uvicorn = prev.uvicorn.overridePythonAttrs (_: {
+          version = "0.25.0";
+          src = fetchFromGitHub {
+            owner = "encode";
+            repo = "uvicorn";
+            rev = "0.25.0";
+            hash = "sha256-ng98DTw49zyFjrPnEwfnPfONyjKKZYuLl0qduxSppYk=";
+          };
+        });
+      })
```

Comment:
Should we perhaps switch the order, _just in case_ someone wants to override `uvicorn`?